girder/api/rest.py

```diff
@@ -20,13 +20,11 @@
 import cherrypy
 import collections
 import datetime
-import functools
 import json
 import pymongo
 import six
 import sys
 import traceback
-import types
```

Comment:
Weird, our style checker usually catches unused imports, do you know why it didn't here?